db/migrate/20200424150635_create_week_day.rb

```diff
@@ -0,0 +1,17 @@
+class CreateWeekDay < ActiveRecord::Migration[6.0]
+  def change
+    create_table :week_days do |t|
+      t.integer :day, null: false
+      t.time :opening_hour
+      t.time :closing_hour
+      t.boolean :active, default: false
+
+      t.string :timestamps
+
+      t.references :store, { foreign_key: { on_delete: :cascade }, index: true }
+
+      t.index :active
+      t.index :day
```

Comment:
I think this index should be `[:store_id, :day], unique: true`. Otherwise the uniqueness validation in the model (`  validates_uniqueness_of :day, scope: :store_id`) won't be backed by a database constraint, potentially leading to inconsistent data. Rubocop tends to complain about this. Not sure why it didn't. I may be getting something mixed up.

Reply:
Ok, I'll do that.